### PR TITLE
Replace legacy TestKit test IDs

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: packages/testkit-backend/src/skipped-tests/common.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ const skippedTests = [
     ifEquals(
       'stub.iteration.test_result_list.TestResultList.test_tx_func_result_list_pulls_all_records_at_once_next_before_list'
     ),
-    ifEquals(
+    ifEquals(test_disconnect_on_commit
       'stub.iteration.test_result_list.TestResultList.test_session_run_result_list_pulls_all_records_at_once_next_before_list'
```
Comment (bigmontz) [inline packages/testkit-backend/src/skipped-tests/common.js]: It didn't seem to be a valid javascript construction and also we won't have any test with full name equal to `test_disconnect_on_commit`

**Review by bigmontz — CHANGES_REQUESTED** [review]
It might have a bug here

File: packages/testkit-backend/src/skipped-tests/common.js (hunk 1 above)
Comment (robsdedude) [reply packages/testkit-backend/src/skipped-tests/common.js]: shame on me

**Review by bigmontz — APPROVED** [review]
🐞

**Pull request merged**